extensions/PhabBugz/lib/User.pm

```diff
@@ -199,5 +200,17 @@ sub get_phab_bugzilla_ids {
   return \@results;
 }
 
-1;
+#################
+#   Helpers     #
+#################
 
+sub set_user_enabled_status {
+  my ($self, $enabled) = @_;
+  my $params = {
+    objectIdentifier => $self->phid,
+    transactions     => [{type => 'disabled', value => ($enabled ? false : true),}]
```

Comment:
Can you do `!$enabled` here?

Reply:
Not sure how that would translate JSON wise. The Phabricator API requires an explicit true or false JSON value and not a 1 or 0. Perl does support using !$enabled. I will try it out.

Reply:
So it has to be set to JSON true or false which is two function provided by the Mojo::JSON library. So I will leave this as is. Anything else I get "Exception when processing transaction of type "disabled": Error while reading "value": Expected boolean (true or false), got something else." errors from Phabricator.

Reply:
Oh, this `true` and `false` was from a _third party library_?
Ok, carry on :)